docs/api/api/api.txt

```diff
@@ -1985,6 +1985,7 @@ XmlResult: status_ok
 
 <staging_workflow_project>: The name of the staging workflow project, e.g. home:user1
 <staging_project>: The name of the staging project, e.g. home:user1:Staging:A
+<request>: The number of the request, e.g. 12345
```

Comment:
Careful. everwhere else in this document we talk about request ids when it talks about the user given number - and about 'number' if we mean a count of requests.